ts/components/conversation/media-gallery/DocumentListItem.tsx

```diff
@@ -0,0 +1,82 @@
+/**
+ * @prettier
+ */
+import React from 'react';
+
+import moment from 'moment';
+import formatFileSize from 'filesize';
+
+interface Props {
+  fileName?: string;
+  fileSize?: number;
+  i18n: (key: string, values?: Array<string>) => string;
+  onClick?: () => void;
+  timestamp: number;
+}
+
+const styles = {
+  container: {
+    width: '100%',
+    height: 72,
+    borderBottomWidth: 1,
+    borderBottomColor: '#ccc',
+    borderBottomStyle: 'solid',
+  },
+  itemContainer: {
+    display: 'flex',
+    flexDirection: 'row',
+    flexWrap: 'nowrap',
+    alignItems: 'center',
+    height: '100%',
+  } as React.CSSProperties,
+  itemMetadata: {
+    display: 'inline-flex',
+    flexDirection: 'column',
+    flexGrow: 1,
+    flexShrink: 0,
+    marginLeft: 8,
+    marginRight: 8,
+  } as React.CSSProperties,
+  itemDate: {
+    display: 'inline-block',
+    flexShrink: 0,
+  },
+  itemIcon: {
+    flexShrink: 0,
+  },
+  itemFileSize: {
+    display: 'inline-block',
+    marginTop: 8,
+    fontSize: '80%',
+  },
+};
+
+export class DocumentListItem extends React.Component<Props, {}> {
+  public renderContent() {
+    const { fileName, fileSize, timestamp } = this.props;
+
+    return (
+      <div style={styles.itemContainer} onClick={this.props.onClick}>
+        <img
+          src="images/file.svg"
+          width="48"
```

Comment:
I feel like this needs to be defined somewhere. All thumbnails the same size. In my outstanding quoted replies PR, I increased the size of Android quote thumbnails to 50px.

Reply:
Interesting. All these icons are natively `48px`: https://github.com/signalapp/Signal-Desktop/blob/89d3078e2a341e55834964ca6847db3fd04b0a09/images/menu.svg#L1

Did you change the native size or just the output size. I realize these are SVG so we can scale them. Happy to do whatever will make it most consistent.